### PR TITLE
feat: Added Grids

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the variations of grids of 2x2 and 3x3
Screenshot (433)

Screenshot (435)

### CODE REVIEW

**Review by Hardiegogo — APPROVED** [review]
Clean.✔

File: CSS/components/grids.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: 1fr 1fr ;
+    grid-template-columns:  1fr 1fr;
+}
+.grid-wrap-3-3{
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: 1fr 1fr  1fr;
+    grid-template-columns:  1fr 1fr 1fr;
```
Comment (sadathshariff) [inline CSS/components/grids.css]: Just s suggestion to use repeat(3,1fr) here

File: CSS/components/grids.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.grid-wrap-2-2{
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: 1fr 1fr ;
+    grid-template-columns:  1fr 1fr;
+}
+.grid-wrap-3-3{
+    display: grid;
```
Comment (sadathshariff) [inline CSS/components/grids.css]: You can take out display grid make that as utility class and use it

**Review by sadathshariff — COMMENTED** [review]
Code looks clean have given some suggestions

File: CSS/components/grids.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/components/grids.css]: Thank you for pointing out , refactored the code 👍

File: CSS/components/grids.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    grid-auto-flow: column;
+    grid-template-rows: repeat(3, 1fr);
+    grid-template-columns:  repeat(3, 1fr);
+}
+.grid-pills{
+    display: flex;
+    justify-content: center;
```
Comment (shuntzhere) [inline CSS/components/grids.css]: You might wanna give it a standard name, grid-pills seems confusing. Also try implementing base classes.

**Review by shuntzhere — COMMENTED** [review]
Suggested to change the name of a class, also try to use base classes for repeating blocks of code.

File: CSS/components/grids.css (hunk 3 above)
Comment (krishanu2911) [reply CSS/components/grids.css]: thank you for pointing out, done added a util class and removed this whole class.

File: CSS/components/grids.css (hunk 2 above)
Comment (krishanu2911) [reply CSS/components/grids.css]: created and  Added the util class for display: grid

**Review by sadathshariff — APPROVED** [review]

**Review by shuntzhere — APPROVED** [review]
Looks good for now, just try to stay consistent across your CSS now that you have named it maple.

**Pull request merged**